iroh-net/src/magicsock.rs

```diff
@@ -1699,10 +1657,17 @@ impl Actor {
         debug!("link change detected: major? {}", is_major);
 
         if is_major {
-            // Clear DNS cache
             DNS_RESOLVER.clear_cache();
             self.inner.re_stun("link-change-major");
-            self.rebind_all().await;
```

Comment:
I think I prefer having this in a method

Reply:
you are not calling `update_net_info` on this path anymore, is that an issue?

Reply:
Moved it to a method.

And good spot!  I did indeed remove that, because this is already done by the `re_stun()` call just above.  In fact that method should not have been allowed to be called, I've documented that in the docstring for now.